### DISCUSSION TITLE
Configuration-Question: hide 'no items to show yet' / empty container

### DISCUSSION THREAD

**Discussion by kimpenhaus** [Lissy93/dashy#444] [Help and Support] [answered]
Hi,

there seems to be a change lately which shows a 'no items to show yet' label in  empty, expanded container with no elements. Before those container were hidden - is there any config prop to enable the old behavior again?

Thanks,
m.

<img width="1326" alt="image" src="https://user-images.githubusercontent.com/575109/150966288-4a1c3566-6abf-4605-b58d-9b694b1a73d0.png">

**Comment by lissy93** [comment, marked as answer]
That does indeed appear to be a bug, sorry about that.
I've opened an issue  (#447) and will include a fix within the next release :)
